### PR TITLE
Feature grid enhancements

### PR DESCRIPTION
**Pull request by dnlkoch**
### Description:
This adds some features to the `FeatureGrid`:
  * Highlight and select features on map if associated rows are hovered or clicked.
  * Remove `attributeMapping` prop, same can be achieved with existing `colDefs` prop (see example).
  * Add example for usage with a remote feature source (including remote paging, filtering and sorting).
  * Remove unused less variables (see style.js).
  * Use `url-loader` to include image files in example builds.
  * Add tests for `FeatureGrid`

### CODE REVIEW

**Comment by marcjansen** [comment]
awesome work. this is a breaking change, right? it removes a prop, doesn't it?

File: src/Grid/FeatureGrid/FeatureGrid.jsx [hunk 1]
```diff
@@ -26,11 +27,32 @@ import './FeatureGrid.less';
 export class FeatureGrid extends React.Component {
 
   /**
-   * The className added to this component.
+   * The className added to add to this component.
```
Comment (marcjansen) [inline src/Grid/FeatureGrid/FeatureGrid.jsx]: remove `added`

File: src/Grid/FeatureGrid/FeatureGrid.jsx [hunk 2]
```diff
@@ -102,29 +136,38 @@ export class FeatureGrid extends React.Component {
     zoomToExtent: PropTypes.bool,
 
     /**
-     * Whether rows should be selectable or not.
+     * Whether rows and features should be selectable or not.
      *
      * @type {Boolean}
      */
     selectableRows: PropTypes.bool,
```
Comment (marcjansen) [inline src/Grid/FeatureGrid/FeatureGrid.jsx]: should we rename to selectable? now that it not only means rows?

File: src/Grid/FeatureGrid/FeatureGrid.jsx [hunk 3]
```diff
@@ -372,16 +592,15 @@ export class FeatureGrid extends React.Component {
   /**
    * Returns the correspondig feature for the given table row key.
    *
-   * @param {String|Number} key The key to get the obtain the feature from.
+   * @param {Number} key The key to get the obtain the feature from.
```
Comment (marcjansen) [inline src/Grid/FeatureGrid/FeatureGrid.jsx]: I suggest to reword the docs

File: src/Grid/FeatureGrid/FeatureGridWfs.example.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: basic.hbs
+title: FeatureGrid WFS example
+description: This example demonstrates the usage of the FeatureGrid to show data from a remote source (WFS GetFeature).
+collection: Examples
+---
+
+This example demonstrates the usage of the FeatureGrid with a remote feature source.
```
Comment (KaiVolland) [inline src/Grid/FeatureGrid/FeatureGridWfs.example.md]: As the example is pretty large you could add some more description.

File: src/Grid/FeatureGrid/FeatureGrid.jsx (hunk 1 above)
Comment (dnlkoch) [reply src/Grid/FeatureGrid/FeatureGrid.jsx]: Fixed in 84bbd2c.

File: src/Grid/FeatureGrid/FeatureGrid.jsx (hunk 2 above)
Comment (dnlkoch) [reply src/Grid/FeatureGrid/FeatureGrid.jsx]: Fixed in 495d1e8.

File: src/Grid/FeatureGrid/FeatureGrid.jsx (hunk 3 above)
Comment (dnlkoch) [reply src/Grid/FeatureGrid/FeatureGrid.jsx]: Fixed in 84bbd2c.

File: src/Grid/FeatureGrid/FeatureGridWfs.example.md (hunk 4 above)
Comment (dnlkoch) [reply src/Grid/FeatureGrid/FeatureGridWfs.example.md]: Fixed in 320b39b.

**Comment by dnlkoch** [comment]
Thanks for your reviews @marcjansen & @KaiVolland!

I addressed all mentioned issues and will merge now.

**Comment by marcjansen** [comment]
👍

**Pull request merged**